docs/Operators.md

```diff
@@ -7509,15 +7509,15 @@ expect(node, inputs=[x], outputs=[y],
       }
   
       graph body-net (
-        %i[INT32, scalar]
-        %keepgoing[BOOL, scalar]
-        %b[INT32, scalar]
+        %i[INT32, scalar]           // iteration number
+        %keepgoing_in[BOOL, scalar] // incoming loop-termination-condition; not used
+        %b_in[INT32, scalar]        // incoming value of loop-carried-dependency b
       ) {
-        %my_local = Add(%a, %b)
-        %b_out = Sub(%a, %b)
-        %keepgoing_out = Greater(%my_local, %b_out)
-        %user_defined_vals = Add(%b, %b)
-        return %keepgoing_out, %b_out, %user_defined_vals
+        %my_local = Add(%a, %b_in)
+        %b_out = Sub(%a, %b_in) // outgoing value of loop-carried-dependency b
```

Comment:
There is no `b` anymore. This statement doesn't explicitly say that `the last value of b_out in one iteration will be the value of b_in in the next iteration`.

Reply:
That's what the pseudo-code is for, to show how b_out is copied into b_in that assignment statement. Explanatory comments may be more useful outside, in the description. I will see if I can add any more to clarify things there.